docs/faq.rst

```diff
@@ -429,7 +429,7 @@ BORG_FILES_CACHE_TTL to at least 26 (or maybe even a small multiple of that),
 it would be much faster.
 
 Another possible reason is that files don't always have the same path, for
-example if you mount a filesystem without stable mount points for each backup.
+example if you mount a filesystem without stable mount points for each backup or if you are running the backup from a filesystem snapshot whose name is not stable.
```

Comment:
in case someone else is wondering about "whose" here, it is correct:

http://dict.leo.org/forum/viewGeneraldiscussion.php?idThread=1086474&idForum=4&lang=de&lp=ende